pages/projects.js

```diff
@@ -0,0 +1,357 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import Checkbox from '@material-ui/core/Checkbox';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import DeleteIcon from '@material-ui/icons/Delete';
+import FilterListIcon from '@material-ui/icons/FilterList';
+import Switch from '@material-ui/core/Switch';
+import Link from '@material-ui/core/Link';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  toolbar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  leftToolbar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  },
+});
+
+function createData(id, name, shared, teamMembers, updatedAt, createdAt) {
+  return {
+    id,
+    name,
+    shared,
+    teamMembers,
+    updatedAt,
+    createdAt,
+  };
+}
+
+const rows = [
```

Comment:
If this is mock data, I think we should comment that, so it is clear during refactor.

Reply:
done